scripts/bulk-access-control.md

```diff
@@ -71,12 +70,11 @@ The json file specified with the file parameter has the following structure
    }
 }
 ```
-* target: is a list of uuid of communities, collections and individual items to process
 * item: the item node contains the processing detail at the item level for the identified resources
   * mode: can be replace or add. Replace will mean that after processing the items will have only the access conditions that are specified in the accessContitions property. Add will mean that after processing the items will have the specified accessCondition in addition to the ones already defined.
   * accessConditions: the list of access conditions to apply, with their specific parameters if needed (such as a startDate for an embargo, etc.). Only in replace mode the accessConditions can be empty in this case the access conditions of the item will be reset to what is inherited from their owningCollection. Please note that when accessConditions is set and is not empty not inheritence from the owningCollection occur. For more information on accessConditions see the [What is an Access Condition?](#what-is-an-access-condition) section below.
-* bitstreams: the bitstream node contains the processing detail at the bitstream level for the identified resources
-  * constraints: only when the `target` element (see above) contains just a single Item UUID, then the 'constraints' can be used to limit which Bitstreams are processed in that Item. If `constraints` specifies one or more bitstream UUIDs, then only those bitstreams listed will be processed. If the `constraint`s node is not present, is empty or contains a null uuid or 0-size list, then all the bitstreams will be processed.
+* bitstreams: the bitstreams node contains the processing details at the bitstream level for the identified resources, applies to bitstreams in the main content bundle (i.e. ORIGINAL bundle) and also applies to derived bitstreams created by the filter-media will be updated according to the changes performed to the corresponding original bitstream expect than for bitstreams derived by public filters classes that will stay with their original policies (set to READ Anonymous at the creation time)
```

Comment:
I don't understand this part. It sounds like it says the permissions of the derived bitstreams are modified as well; AND it that the permissions of the derived bitstreams remain unchanged. Can you specify which exactly is happening?

Reply:
@benbosman  what about 

the bitstreams node contains the processing details at the bitstream level for the identified resources. The changes will be applied to bitstreams in the main content bundle (i.e. ORIGINAL bundle) and to all the corresponding derived bitstreams created by the filter-media except than for bitstreams derived by public filters classes that will stay with their original policies (set to READ Anonymous at the creation time)

Reply:
Here's slightly clearer English which I think is saying the same thing...

the `bitstreams` node contains the processing details at the bitstream level for the identified resources.  These changes will be applied to bitstreams in the main content bundle (i.e. ORIGINAL bundle) and to any corresponding, derivative bitstreams created by `filter-media`.  The only exception is that any derivative bitstreams created with `publicPermission` (i.e. set to READ Anonymous at creation via `filter.org.dspace.app.mediafilter.publicPermission`) will keep their existing public policies.